### DISCUSSION TITLE
現状の砂漠の大体の完成度

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#72] [エリア・建築の完成度・開発状況]
1. 上層名：追憶の海
- [x] 岩内部の建築
_砂漠の主要な数箇所は建築が完了している状態。_
- [x] 初期地点のゲート周辺
_開拓済み。_
- [x] 小ダンジョン
_いくつか配置済み。_
- [x] 上空のいくつかの建築
_月のゲートの完成がまだ。また浮遊する逆ピラニット内部の開発を再度行う模様_
- [x] 砂漠外の岩のリング、及び中央大陸外の４つの浮島。
_余裕があればネタを入れたいところ。４つのうち１つが未完成。_

2. 下層名：微睡の海溝
- [x] いくつかの小ダンジョン
_数箇所に設置。_
- [x] 下層へと続く穴の開拓
_満足する出来栄え。_
- [x] 過去を思わせる遺物の配置
_しょぼ氏等ががんばった結果相当数は確保。_
- [x] 下層全域における修正
_主におかしいブロックや穴等の修正。見える範囲は全て完了している。_
- [ ] 酸床における理不尽ポイントの確認と修正
_後々行うつもりではある。_

**要望**
- [x] 中央大陸外のひかるつ氏の製作中の古代王宮の下部土台から見えている地下建築の箇所を埋めて整えて欲しい。

**Comment by anminmakura** [comment]
天ぷらうどん氏が砂漠の朽ち果てた斜塔、air氏の隠された遺跡の真上に
飛空挺、離陸場、研究施設を建築。

**Comment by anminmakura** [comment]
**ひかるつ氏進捗まとめ。**
古代王宮(旧名:太陽の遺跡)の天井と床の装飾・光源の設置を進行中。
島の上面の建築も着手。
![古代王宮01](https://user-images.githubusercontent.com/39670701/98611329-12c3e900-2335-11eb-9089-f4413b6a4d1d.png)
![古代王宮02](https://user-images.githubusercontent.com/39670701/98611356-1f484180-2335-11eb-966f-bb099e35703a.png)
![古代王宮03](https://user-images.githubusercontent.com/39670701/98611388-2cfdc700-2335-11eb-9c99-7f202203cdb7.png)
![古代王宮04](https://user-images.githubusercontent.com/39670701/98611414-3a1ab600-2335-11eb-992e-9918a9244fce.png)

**Comment by anminmakura** [comment]
**ひかるつ氏進捗まとめ**。
ピラミッドの外装を作成。
色々やって内装には干渉しないように編集。
![ピラミッド外装](https://user-images.githubusercontent.com/39670701/98611500-620a1980-2335-11eb-9075-690b5c6dc5f6.png)

**Comment by anminmakura** [comment]
**ひかるつ氏進捗まとめ。**
他建築班の追加装飾も待ってます。
![古代王宮05](https://user-images.githubusercontent.com/39670701/98611593-88c85000-2335-11eb-8a06-4c54201b8728.png)
![古代王宮06](https://user-images.githubusercontent.com/39670701/98611614-91208b00-2335-11eb-9c98-212b03c76601.png)
古代王宮内部の教会?に試験的にオーディエンスも設置。
古代王宮の進捗が80%が100%になりつつある。
古代王宮の大臣の部屋(というより、通路？)を作成。
![古代王宮07](https://user-images.githubusercontent.com/39670701/98611650-a85f7880-2335-11eb-92c0-0a8cd45e9e54.png)
![古代王宮08](https://user-images.githubusercontent.com/39670701/98611652-a990a580-2335-11eb-8a88-c9d2e7e0ac9b.png)

**Comment by anminmakura** [comment]
ひかるつ氏が古代王宮内部のクオリティーを底上げ中。
![まとめ3](https://user-images.githubusercontent.com/39670701/99180506-1955c480-276a-11eb-9807-22bc0ff24134.png)
![まとめ2](https://user-images.githubusercontent.com/39670701/99180509-1a86f180-276a-11eb-9ed7-8382df16023f.png)
![まとめ1](https://user-images.githubusercontent.com/39670701/99180512-1bb81e80-276a-11eb-8c35-0cec28d22753.png)

**Comment by HikaruTsu** [comment]
王の間まで続く回廊について、壁の進捗を進めました。

ここの壁の封印の案として
・壁にスポナーを埋め込んで中BOSS級のちょっとつよめのMOBを一回だけスポーンさせる（ドラクエ３のゾーマ戦的な感じ）
・スポナーが破壊されると同時にそこの壁がﾊﾟｧﾝと派手に砕け散る
or
・各地に設置されたボタンを押して封印をﾊﾟﾘﾝっと解除する
を提案してみる
でもぶっちゃけ他の案でもいいです
あとこういう仕組み作るのめんどくさいから作ってくれる人がほしいです

下の溶岩はバリアブロックがあるから入れない感じになりました(透明床として)
なので溶岩には入れないけど、一つだけ穴があってそこからエンパで入ることができます
ついでに言うと、そこからさらに隠し穴にエンパを投げ込むと壁をスルーして王の間に行けるようになっています
(どうしても壁を突破したくない人向け&不評なら潰す)
まぁこれについては話すだけだとよくわかんないのでわかんなかったら私に教えてって言えばどういうことか説明します

![image](https://user-images.githubusercontent.com/54533354/99180702-faf0c880-276b-11eb-8af4-abc3e096ebf4.png)

**Comment by HikaruTsu** [comment]
色々光源を点けて明るくしました
![image](https://user-images.githubusercontent.com/54533354/99180735-4b682600-276c-11eb-8762-0b6cbb5b71f0.png)

**Comment by HikaruTsu** [comment]
隠し部屋を設置
多分ここがダンジョン内では一番y座標的に高いとこになります
ちなみに地上部分からうっすら見えるポイントがあります
![image](https://user-images.githubusercontent.com/54533354/99180743-558a2480-276c-11eb-816c-ab40475a1e08.png)

**Comment by anminmakura** [comment]
https://user-images.githubusercontent.com/39670701/129450454-042c78c1-29c6-4acf-8562-24a47dc76781.mp4


A、砂漠地下侵入経路(微睡の海溝(バヌム＝ンヴァズラ内部))への入口
- [x] 1、-1930 164 2388　鋭岩の道
- [x] 2、-2000 98 2325　研究所防衛ゲード出入口
- [x] 3、-2059 112 2232　研究所内部廃棄口
- [ ] 4、-1896 74 2735　古代神殿最深部入口(一方通行)
- [x] 5、-1892 27 2815　バヌム＝ンヴァズラの口
※6
※7

B、超古代神殿(ムーンゲート開放条件)
- [x] 1、-1647 131 2180　太陽の神殿内部　第一の封印
- [x] 2、-1860 38 2455　 微睡の海溝内部　第二の封印
- [x] 3、-2152 110 2622　砂金の塔　第三の封印
- [x] 4、-2265 26 2669　微睡の海溝内部　第四の封印
- [x] 5、-2166 59 2232　微睡の海溝内部　逆さ神殿　第五の封印
- [x] 6、-1944 140 2529　破壊された神殿　第六の封印


## 砂漠地上　追憶の海

- [x] 01、黒の探検隊のコンビナート(砂漠初期地点ゲート) -1830 45 2393

- [x] 02、気球船空港所 -2052 129 2325

- [x] 03、斜塔の隠された地下遺跡 -2155 121 2329

- [x] 04、IROHA遺跡 -2269 130 2387

- [x] 05、太陽の塔 -2275 109 2576

- [x] A6、生贄の祭壇の遺跡 -2273 106 2716 ※A6

- [x] A7、ゴーレム王の神殿 -2078 107 2809 ※A7

- [x] 08、古代王家の空中水園 -2077 156 2682

- [x] 09、古代一般人の生活拠点遺跡 -1855 134 2524

- [x] 10、ムーンゲート入口 -1999 242 2461

- [ ] 【重要】11、ムーンゲート内部からの出口 -1945 140 2530 ※B6

- [x] 12、破壊された古代遺跡 -1880 114 2701

- [x] 13、埋もれたピラミッド -1749 139 2573

- [x] 14、太陽の神殿(離島A) -1682 145 2272

- [x] 15、隠された砂漠の銭湯(離島B) -2412 221 2223

- [x] 16、大岩と骨の島の地下迷宮(離島C) -2464 129 2950

- [x] 17、オアシスと水の遺跡(離島D) -1704 161 2869

- [x] 18、飛行中の気球船 -1627 223 2797

## 砂漠地下　微睡の海溝

- [x] 01(11)、ムーンゲート内部(B6)の地下) -1954 96 2531

- [x] 02、飲み込まれた古代人の遺跡 -1929 35 2352

- [x] 03、虫歯遺跡 -2088 37 2863

- [x] 04、隠された宝箱の遺跡 -2357 73 2436
